src/ccl/ccl.component.ts

```diff
@@ -3,16 +3,19 @@
  * App Component
  */
 import { Component, ViewEncapsulation } from '@angular/core';
+import { Sidebar } from './sidebar/sidebar.component';
+import { Chart } from './charts/chart.component';
```

Comment:
Since you added an index file for this folder, you should be able to just do `import { Chart } from './charts/';`